index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Nice! Love that we're including a language specification 👍

Reply:
Isn't this so nice?